tempodb/backend/block_meta.go

```diff
@@ -29,6 +29,7 @@ type BlockMeta struct {
 	TotalRecords    uint32    `json:"totalRecords"`    // Total Records stored in the index file
 	DataEncoding    string    `json:"dataEncoding"`    // DataEncoding is a string provided externally, but tracked by tempodb that indicates the way the bytes are encoded
 	BloomShardCount uint16    `json:"bloomShards"`     // Number of bloom filter shards
+	FooterSize      uint32    `json:"footerSize"`      // Size of data file footer (parquet)
```

Comment:
Does this not have to be propagated down the search path from query frontend to querier to serverless?

Reply:
It technically will work without it, the `parquetOptimizedReader` skips over that optimization when FooterSize=0, and it will hit the backend.  Good catch though, it's best to include it.